### ISSUE TITLE
Add a How To section

### ISSUE THREAD

**Issue by ddunkin** [FacilityApi/FacilityApi.github.io#19]
Add a section that shows how to model different APIs in FSD with examples. There are many examples in https://github.com/FacilityApi/Facility/tree/master/example, but they aren't easy to browse. Some how-tos could be:
* How to model different responses for different HTTP status codes
* How to model [redirects](https://github.com/FacilityApi/Facility/issues/21)
* How to model HTTP caching

**Comment by ejball** [comment]
Absolutely; we could use a lot more and better documentation. Unfortunately, given the choice between writing code and writing docs, I always seem to choose the code. :-)

But specific suggestions like this are very helpful; thanks.
